oauth2server/proj/settings/local.py

```diff
@@ -2,6 +2,12 @@
 
 from proj.settings.default import *
```

Comment:
I don't think you can import `proj` like this from here I don't think, this needs to be `from oauth2server.proj.settings.default import *`

Reply:
^ I take this one back after running the django server but I don't understand how it works :S